fix: minor fixes around auto height (#645)
appsmithorg/appsmith-docs: website/docs/core-concepts/building-ui/dynamic-ui/README.md

![Click to expand](</img/form_-_table.gif>)

## Setting widget height

> You can set the height of widget using the Height property. It configures how a widget’s height reacts to content changes in the app. To build a dynamic UI, you can use the Auto Height, that gives the widget a capability to change height in response to content changes. Auto height saves you from the task of defining the height of the widget manually. Click [here](/reference/widgets/#auto-height) to read more about Auto height. 

Reviewer: Add hyperlink to the actual content. 
```suggestion
You can set the height of widget using the Height property. It configures how a widget’s height reacts to content changes in the app. To build a dynamic UI, you can use the Auto Height, that gives the widget a capability to change height in response to content changes. Auto height saves you from the task of defining the height of the widget manually. For more information, see the [Auto height](/reference/widgets/#auto-height) property of widgets . 
```
